### PR TITLE
array koans

### PR DESCRIPTION
**Pull request by camilo1704**

### CODE REVIEW

File: koans/ArrayKoans.jl [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
+function array_1d(n)
+    # return a 1D array of zeros with length n
+end
+
+function array_2d()
+    # return a 2x2 matrix of zeros
+end
+
+function array_shape(array)
+    # return the shape of the array
```
Comment (igaray) [inline koans/ArrayKoans.jl]: Some examples showing what exactly the function should return would help guide the user.

Comment (igaray) [reply koans/ArrayKoans.jl]: Koans should: 
- [ ] Have a description of the problem to solve. In this case, we are trying to write a function which will answer the question "what shape does the given array have?", if possible it would be interesting the koan exercise should include an example of what motivates this problem.
- [ ] the solution should link to the documentation for the function used.

File: tests/tests.jl [hunk 2]
```diff
@@ -53,7 +53,36 @@ using ConversionKoans
 end
 
 using ArrayKoans
-@testset "Arrays" begin
+@testset "Array Koans" begin
```
Comment (igaray) [inline tests/tests.jl:56]: A useful criteria for finding test specification boundaries is: 0, 1, 3, n, outside domain.

Comment (igaray) [reply tests/tests.jl:56]: Add some tests trying to find edge conditions and behavior boundaries.

**Review by igaray — CHANGES_REQUESTED** [review]

**Review by igaray — APPROVED** [review]
